packages/core/lib/commands/install.js

```diff
@@ -3,33 +3,43 @@ const command = {
   description: "Install a package from the Ethereum Package Registry",
   builder: {},
   help: {
-    usage: "truffle install <package_name>[@<version>]",
+    usage: "truffle install <packageId> [--alias]",
     options: [
       {
-        option: "package_name",
-        description:
-          "Name of the package as listed in the Ethereum Package Registry. (required)"
+        option: "packageId",
+        description: `(required) Name of the package as listed in the Ethereum Package Registry. 
+          Accepted formats: packageName, packageName@version, ethpm URI, ipfs URI.`
       },
       {
-        option: "<@version>",
+        option: "--alias",
         description:
-          "When specified, will install a specific version of the package, otherwise " +
-          "will install\n                    the latest version."
+          "A different name under which this package will be installed."
       }
     ]
   },
-  run: function(options, done) {
+  run: async function (options, done) {
```

Comment:
When testing locally, I occasionally run into edge cases that hang indefinitely. Is there an existing mechanism to handle these cases? Would it be a good idea to implement some kind of timeout limit when running an ethpm command?

Reply:
This might be because you're making a callback style function also async. Generally not a good idea to do that - use one or the other (see other comment about `callbackify`)